sdk/formrecognizer/Azure.AI.FormRecognizer/src/DocumentTable.cs

```diff
@@ -11,11 +10,9 @@ namespace Azure.AI.FormRecognizer.DocumentAnalysis
     public partial class DocumentTable
```

Comment:
>DocumentTable [](http://example.com/codeflow?start=25&length=13)

Why internal?

Reply:
We're hiding anything related to `DocumentCaption` and `DocumentFootnote` classes since they'll be removed in the next service version. I could wait for the swagger updates to do that, but I wanted to have it reflected in the apiview soon to present the design to architects.

Reply:
We need to remind Paul to remove this from the swagger before GA